app/ui/src/main/java/com/fsck/k9/fragment/MessageListAdapter.kt

```diff
@@ -32,7 +32,8 @@ class MessageListAdapter internal constructor(
         private val res: Resources,
         private val fragment: MessageListFragment,
         private val layoutInflater: LayoutInflater,
-        private val messageHelper: MessageHelper
+        private val messageHelper: MessageHelper,
+        private val showingThreadedList: Boolean = false
```

Comment:
This should also go into the `MessageListAppearance` class.

Reply:
done.